db/seeds.rb

```diff
@@ -5,3 +5,9 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+lifehacks = Lifehack.create(
+  [{title: 'How to tie a tie',
+    description: 'tie a knot'},
+    {title: 'How to tie a tie1'}]
```

Comment:
Align the elements of an array literal if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.